scripts/stack-manager/generate.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```

Comment:
Move all stack-manager scripts into stack manager cloud repo, published as a version artifact (along with the lambda functions).
The stack manager creation playbook should then download the artifact, unpack, and prepare the staged files accordingly.